R/example.R

```diff
@@ -195,4 +195,43 @@ example_large_sized_commits <- function(folder_path="/tmp", folder_name) {
 }
 
 
+#' Create sample mbox with three replies
+#'
+#' Creating a fake mbox with three replies. The first two fake mbox replies have the same email subject while the
+#' third reply has a different subject.
+#'
+#' @param folder_path Default folder path set to "/tmp"
+#' @param folder_name Name of folder where .mbox file will be stored
+#' @return Folder path of .mbox sample file that was created
+#' @export
+example_mbox_three_replies <- function(folder_path = "/tmp", folder_name) {
+
+  # Define sample data for create_fake_mbox function that can easily be altered for testing
+
+  # Step 1: Create fake mbox replies and assign them to variables for easy editing
+  reply_1 <- create_fake_mbox_replies("test-list", "John Doe", "johndoe@example.com", "Janette Doe", "janedoe@example.com",
+                                      "Smithsonian Doe", "smith_doe@example.com", "2023-01-15T08:30:00", "EST",
+                                      "Sample Email Subject ONE", "This is the body of the test email 1.")
+
+  reply_2 <- create_fake_mbox_replies("test-list", "John Doe", "johndoe@example.com", "Janette Doe", "janedoe@example.com",
+                                      "Smithsonian Doe", "smith_doe@example.com", "2023-01-15T09:45:00", "EST",
+                                      "Sample Email Subject ONE", "This is the body of the test email 2.")
+
+  reply_3 <- create_fake_mbox_replies("test-list", "John Doe", "johndoe@example.com", "Janette Doe", "janedoe@example.com",
+                                      "Smithsonian Doe", "smith_doe@example.com", "2023-01-15T11:52:00", "EST",
+                                      "Sample Email Subject TWO", "This is the body of the test email 3.")
+
+  # Step 2: Add each reply to a list
+  replies <- list(reply_1, reply_2, reply_3)
```

Comment:
One minor change here (mistake was mine): Instead of calling `list`, use `c` instead. So:

 replies <- c(reply_1, reply_2, reply_3)

I forgot you were just creating a string variable, and for that, a array of strings (which is defined by `c`) is more than sufficient. Please adjust the parameter documentation accordingly.